packages/dcos-integration-test/extra/test_composition.py

```diff
@@ -87,9 +87,9 @@ def test_signal_service(dcos_api_session):
     if enabled == 'false':
         pytest.skip('Telemetry disabled in /opt/mesosphere/etc/dcos-signal-config.json... skipping test')
 
-    logging.info("Version: ", dcos_version)
```

Comment:
Glad you didn't program in C, coz' that's the only way to use in `printf`.

The other style that I prefer (both in Python and Java) is

1) Create a logger with formatter (one time in common util class).


```
def get_logger():
    logger = logging.getLogger(name='mergebot')
    handler = logging.StreamHandler()
    formatter = logging.Formatter('%(asctime)s %(name)-12s %(levelname)-8s %(message)s')
    handler.setFormatter(formatter)
    logger.addHandler(handler)
    logger.setLevel(logging.DEBUG)
    return logger


log = get_logger()
```

2) Use .formatted string.

```
log.debug("Failed parsing r{}".format(config_file_contents))
```